scripts/apps/contacts/components/ItemContainer.tsx

```diff
@@ -74,7 +74,7 @@ export class ItemContainer extends React.Component<any, any> {
         }
 
         if (item.country) {
-            addressInfo.push(item.country);
+            addressInfo.push(item.country?.name);
```

Comment:
If it doesn't have a `name` property, `undefined` will be pushed which would probably not desired behavior. Modify the condition above instead to check if name exists.

```
if (item.country?.name != null) {
    addressInfo.push(item.country.name);
}
```